boards/arm/frdm_k64f/frdm_k64f_defconfig

```diff
@@ -10,3 +10,6 @@ CONFIG_GPIO=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=120000000
 CONFIG_OSC_EXTERNAL=y
 CONFIG_HAS_DTS=y
+
+# Enable MPU
+CONFIG_HAS_SYSMPU=y
```

Comment:
Do we really want to enable this by default? By disabling execution from SRAM, we'll break anything that uses the kinetis flash driver. For example, mcuboot.